mat/lq.go

```diff
@@ -115,29 +120,33 @@ func (lq *LQ) LTo(dst *Dense) *Dense {
 	dst.Copy(t)
 
 	if r == c {
-		return dst
+		return
 	}
 	// Zero right of the triangular.
 	for i := 0; i < r; i++ {
 		zero(dst.mat.Data[i*dst.mat.Stride+r : i*dst.mat.Stride+c])
 	}
-
-	return dst
 }
 
 // QTo extracts the n×n orthonormal matrix Q from an LQ decomposition.
-// If dst is nil, a new matrix is allocated. The resulting Q matrix is returned.
-// QTo will panic if the receiver does not contain a factorization.
-func (lq *LQ) QTo(dst *Dense) *Dense {
+//
+// If dst is empty, QTo will resize dst to be c×c. If dst is
+// non-empty, QTo will panic if dst is not c×c. QTo will also panic
```

Comment:
"When dst is non-empty, QTo will panic if dst is not c×c."

Reply:
Done.